.vscode/launch.json

```diff
@@ -5,20 +5,15 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": "Debug current test file in VSCode",
             "type": "node",
             "request": "launch",
-            "runtimeArgs": [
-                "--inspect-brk",
-                "${workspaceRoot}/node_modules/jest/bin/jest.js",
-                "--runInBand",
-                "${fileBasename}",
-                "--coverage",
-                "false"
-            ],
+            "name": "Debug current unit test file",
+            "program": "${workspaceFolder:root}/node_modules/jest/bin/jest",
+            // ${relativeFile} returns path with back slash. jest can't find the file if we use back slash. So, just passing file name instead
+            "args": ["${fileBasename}", "--runInBand", "--coverage=false"],
             "console": "integratedTerminal",
             "internalConsoleOptions": "neverOpen",
-            "port": 9229
+            "cwd": "${fileDirname}"
```

Comment:
It seems odd that this file would change. Why is this?

Reply:
I reverted to the previous version b/c it runs multiple test files even they are not open in VS Code